azure-function-app/massdriver.yaml

```diff
@@ -153,13 +153,29 @@ params:
           description: Azure App Service Plan size
           type: string
           oneOf:
-            - title: 1 vCPU, 1.75GB RAM
+            - title: 1 vCPU, 1.75GB RAM, scales up to 3 instances
+              const: B1
+            - title: 2 vCPU, 3.5GB RAM, scales up to 3 instances
+              const: B2
+            - title: 4 vCPU, 7GB RAM, scales up to 3 instances
+              const: B3
+            - title: 1 vCPU, 1.75GB RAM, scales up to 10 instances
               const: S1
-            - title: 2 vCPU 8GB RAM
+            - title: 2 vCPU, 3.5GB RAM, scales up to 10 instances
```

Comment:
Did you provision any of these? I know we've have some sku issues lately and I'm hesitant to take these in blindly ya know. Could I get a Terraform run of a few of them please?

Reply:
Will test as soon as mdxc issue is fixed, currently preventing apps from deploying